packages/main/src/features/ListItemStandardExpandableTextTemplate.tsx

```diff
@@ -0,0 +1,31 @@
+import ExpandableText from "../ExpandableText.js";
+import type ListItemStandard from "../ListItemStandard.js";
+import type { ExpandableTextTemplateParams } from "../types/ExpandableTextTemplateParams.js";
+
+/**
+ * Provides a template for rendering text with the ExpandableText component
+ * when wrappingType is set to "Normal".
+ *
+ * @param {object} options - The configuration options for the expandable text
+ * @returns {JSX.Element} The rendered ExpandableText component
+ */
+export default function ListItemStandardExpandableTextTemplate(
+	this: ListItemStandard,
+	options: ExpandableTextTemplateParams
```

Comment:
to be consistent with other templates, lets rename this prop to injectedProps, check ButtonTemplate or ListItemBaseTempalte